content/post/internal-boot2root-writeup.md

```diff
@@ -0,0 +1,543 @@
+tcp        0      0 127.0.0.53:53           0.0.0.0:*               LISTEN      -                   
+tcp        0      0 0.0.0.0:22              0.0.0.0:*               LISTEN      -                   
+tcp        0      0 10.10.215.86:58660      10.10.253.137:4444      ESTABLISHED 6511/TOJWR          
+tcp        0      0 10.10.215.86:55326      10.10.253.137:4433      ESTABLISHED 6511/TOJWR          
+tcp6       0      0 :::80                   :::*                    LISTEN      -                   
+tcp6       0      0 :::22                   :::*                    LISTEN      -                   
+tcp6       0      0 10.10.215.86:80         10.10.253.137:44196     ESTABLISHED -                   
+tcp6       0      0 10.10.215.86:80         10.10.253.137:44286     TIME_WAIT   -                   
+udp        0      0 127.0.0.53:53           0.0.0.0:*                           -                   
+udp        0      0 10.10.215.86:68         0.0.0.0:*                           - 
+```
+
+```
+docker0: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
+        inet 172.17.0.1  netmask 255.255.0.0  broadcast 172.17.255.255
+        inet6 fe80::42:d6ff:fe0f:a70e  prefixlen 64  scopeid 0x20<link>
+        ether 02:42:d6:0f:a7:0e  txqueuelen 0  (Ethernet)
+        RX packets 8  bytes 420 (420.0 B)
+        RX errors 0  dropped 0  overruns 0  frame 0
+        TX packets 19  bytes 1416 (1.4 KB)
+        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0
+```
+
+```
+================================( Processes, Cron, Services, Timers & Sockets )================================
+[+] Cleaned processes
+[i] Check weird & unexpected proceses run by root: https://book.hacktricks.xyz/linux-unix/privilege-escalation#processes
+USER       PID %CPU %MEM    VSZ   RSS TTY      STAT START   TIME COMMAND
+aubrean+  1506  0.0  0.0   1148     4 ?        Ss   04:08   0:00 /sbin/tini -- /usr/local/bin/jenkins.sh
+aubrean+  1540  0.5 12.1 2587808 248012 ?      Sl   04:08   0:27 java -Duser.home=/var/jenkins_home -Djenkins.model.Jenkins.slaveAgentPort=50000 -jar /usr/share/jenkins/jenkins.war  0.0  0.1  28332  2344 ?        Ss   04:08   0:00 /usr/sbin/atd -f
+message+   909  0.0  0.2  50060  4664 ?        Ss   04:08   0:00 /usr/bin/dbus-daemon --system --address=systemd: --nofork --nopidfile --systemd-activation --syslog-onlyql     1099  0.2 11.2 1165848 229308 ?      Sl   04:08   0:14 /usr/sbin/mysqld --daemonize --pid-file=/run/mysqld/mysqld.pid
+```
+
+I decided to take a look in the `/var/jenkins_home/` directory to see if I could find any configuration for the jenkins host but found that it didn't exist, giving credence to it running in a docker container. I similarly looked under `/etc/` and found nothing related to jenkins, finally I looked under `/opt/` and stumbled on a `/opt/wp-save.txt` file that netted a set of credentialls for the `aubreanna` user.
```

Comment:
, leading me to believe it was running in a docker container

Similarly, I looked under `/etc/`